image.md

```diff
@@ -0,0 +1,140 @@
+# Image
+
+This section defines a format for encoding a container's *filesytem bundle* as a portable *image*.
+An image is a set of files organized in a certain way, and containing all of the necessary data and metadata, for any compliant runtime to verify the validity of the image's contents and to construct a filesystem bundle represented by the image.
+Images are meant to be portable such that they can be used to reliably move filesystem bundles between instances of OCI compliant implementations.
+
+## Notational Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this and other documents of the specification are to be interpreted as described in [RFC2119](http://tools.ietf.org/html/rfc2119).
+
+## Image Contents
+
+A compliant image contains the following pieces of information:
+
+1. An OCI [`config.json`](config.md) file.
+This file is REQUIRED to be present in the image.
+The `config.json` file contains the host independent configuration information necessary to execute the container.
+See [`config.json`](config.md) for more information.
+
+1. The root filesystem of the filesystem bundle.
+This directory MUST be present in the image, however it MAY be empty (not contain any files or folders).
+As specified in the `config.json` file, this directory contains the files that make up the root filesystem of the container.
+This directory MAY contain only a portion of the complete set of files that make up the filesystem bundle, in which case the remaining set of files MUST be found in one of the dependent images.
+
+1. References to dependent images.
+If present, this OPTIONAL information MUST be in a file called `required.json`.
+This file MUST be a json encoded array of references to images as specified in the [Dependent Images](#dependent-images) section.
+
+1. Dependent images.
+An image MAY choose to include dependent images as directories within the image.
+If present, each dependent image MUST be placed in a directory whose name is the same as its ID.
+Within each directory MUST be a complete OCI compliant image definition.
+Note that within each dependent image's directory there MAY be a whiteout.json file.
+
+1. Whiteout list.
+An image MAY choose to include a list of files that are to be considered to be deleted from the root filesystem even though they are included within the image's root filesystem directory.
+It is an implementation detail to decide how this whiteout list is processed but an implementation MUST ensure that any container that is created from this image will behave the same as if those files were not in the root filesystem at all.
+The whiteout list MUST be in a file called `whiteout.json`.
+See the [Whiteout List](#whiteout-list) section for more details.
+
+1. Extension files.
+An image MAY choose to include additional files in the image.
```

Comment:
+1, although I'd say “files and directories” to avoid any uncertainty.  And I'd update bundle.md to say that this sort of thing was permissable in the filesystem bundle (see [this thread][1]), after which my suggested [bundle.md reference][2] would cover this point.

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/6ZKMNWujDhU
[2]: https://github.com/opencontainers/specs/pull/293/files#r48905528